rasterio/_io.pyx

```diff
@@ -660,7 +698,7 @@ cdef class DatasetReaderBase(DatasetBase):
         warnings.warn(
             "read_mask() is deprecated and will be removed by Rasterio 1.0. "
             "Please use read_masks() instead.",
-            FutureWarning,
+            DeprecationWarning,
```

Comment:
No, I want to keep the `DeprecationWarning`. I'm following Brett Slatkin's refactoring guidelines.